threadsd/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 	connLowWater := fs.Int("connLowWater", 100, "Low watermark of libp2p connections that'll be maintained")
 	connHighWater := fs.Int("connHighWater", 400, "High watermark of libp2p connections that'll be maintained")
 	connGracePeriod := fs.Duration("connGracePeriod", time.Second*20, "Duration a newly opened connection is given before it becomes subject to pruning")
+	keepAliveInterval := fs.Duration("keepAliveInterval", time.Second*5, "Enables websocket keepalive pinging with the specified timeout interval. Configured interval must be >= 1s.")
```

Comment:
Leaving this defaulting to 5 seconds, as the Listen API is much more useful when the socket is kept alive (in the case of slow updates in realistic scenarios).